docs/pages/x/api/date-pickers/date-picker.json

```diff
@@ -36,6 +36,10 @@
     "displayWeekNumber": { "type": { "name": "bool" } },
     "fixedWeekNumber": { "type": { "name": "number" }, "default": "undefined" },
     "format": { "type": { "name": "string" } },
+    "formatDensity": {
+      "type": { "name": "enum", "description": "'dense'<br>&#124;&nbsp;'spacious'" },
+      "default": "\"normal\""
```

Comment:
Should it be "normal" or "dense" here? Same question in [date-range-picker.json](https://github.com/mui/mui-x/pull/8425/files#diff-b2cfa2e0a62249da271d6ee4c814a5b7342416c819eb9ce1e34ef3c73c18bce3R51)